common/views/components/Buttons/ButtonButton/ButtonButton.js

```diff
@@ -9,7 +9,7 @@ type Props = {|
   text: string,
   eventTracking?: string,
   id?: string,
-  clickHandler?: () => void
+  clickHandler?: Function
```

Comment:
When would a click handler ever return anything?

Reply:
No idea - I think it's the generic type, which makes me think we could be more specific?